samples/net/sockets/echo_client/overlay-ot-rcp-host-nxp.conf

```diff
@@ -0,0 +1,57 @@
+CONFIG_REQUIRES_FULL_LIBC=y
```

Comment:
One more thing, would be good to add an entry in `sample.yaml` so that we build this configuration for your NXP board:
https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/net/sockets/echo_client/sample.yaml#L111
Same for echo_server.

Reply:
Resolved